File: Jour10/jour10.py
#coding: utf-8
print("============ AFFICHAGE D'INFORMATION ============")
while 1:
	age = input("Donner votre age : ")
	try:
		age = int(age)
	except:
		print("Saisie invalide !")
		continue
	else:
		break
sexe = input("Donner votre sexe (F/M): ")
if((age >= 8 and age <= 14) and sexe == 'M'):
	print(f"Vous etes un adolescent de {age} an(s).")
elif((age >= 8 and age <= 14) and sexe == 'F'):
	print(f"Vous etes une adolescente de {age} an(s).")
else:
	print("Vous n'etes pas d'une tranche d'adolescents .... merci.")

File: Jour10/jour10.c
#include<stdio.h>
#include<conio.h>

int main()
{
	int age;
	char sexe;
	printf(" ========== AFFICHAGE D'INFORMATION =======\n");
	printf("Donner votre age : ");
	scanf("%d",&age);
	//Attention zero control sur age
	

	printf("Donner votre sexe F/M : ");
	while(getchar() != '\n');
	sexe = getchar() ;
	//Attention zero control sur sexe

	if((age >= 8 && age <= 14) && sexe == 'M')
	{
		printf("Vous etes un adolescent de %d an(s).",age);
		 if((age >= 8 && age <= 14) && sexe == 'F')
			printf("Vous etes une adolescente de %d an(s).",age);
	}
	else
		printf("Vous n'etes pas d'une tranche d'adolescents .... merci.");
}